src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/ModelMetadataIdentity.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    public struct ModelMetadataIdentity
```

Comment:
wondering why do we want a struct to represent this ? Is it because this is being used as a key so a value type makes sense?